x-pack/filebeat/docs/inputs/input-cel.asciidoc

```diff
@@ -434,6 +434,26 @@ This specifies SSL/TLS configuration. If the ssl section is missing, the host's
 CAs are used for HTTPS connections. See <<configuration-ssl>> for more
 information.
 
+[float]
+==== `resource.keep_alive.disable`
+
+This specifies whether to disable keep-alives for HTTP end-points. Default: `true`.
+
+[float]
+==== `resource.keep_alive.max_idle_connections`
+
+The maximum number of idle connections across all hosts. Zero means no limit. Default: `0`.
+
+[float]
+==== `resource.keep_alive.max_idle_connections_per_host`
+
+The maximum idle connections to keep per-host. If zero, defaults to two. Default: `0`.
+
+[float]
+==== `resource.keep_alive.idle_connection_timeout`
+
+The maximum amount of time an idle connection will remain idle before closing itself. Zero means no limit. Default: `0s`.
```

Comment:
Maybe we could mention here different units of time this setting could take as a valid input.

Reply:
Done. I've taken the text from `resource.timeout` (CEL0 and `request.timeout` (HTTPJSON).

Reply:
Great, thanks!